src/App.js

```diff
@@ -9,49 +9,95 @@ import { useStateValue } from './store/store';
 import {
 	CTLViewsHoC,
 	Disease,
+	InvalidParameters,
 	Manual,
-	NoListingType,
 	PageNotFound,
 } from './views';
 
 const App = () => {
 	let dynamicRoutes;
 	const { BasePath, CodeOrPurlPath } = useAppPaths();
-	const [{ trialListingPageType }] = useStateValue();
+	const [
+		{ cisBannerImgUrlLarge, cisBannerImgUrlSmall, trialListingPageType },
+	] = useStateValue();
+
+	// Check for image parameters, and set string for invalid parameters page.
+	const missingBannerImages =
+		cisBannerImgUrlLarge == null || cisBannerImgUrlSmall == null;
+	const imageParams = 'cisBannerImgUrlLarge, cisBannerImgUrlSmall';
 
 	const WrappedDisease = CTLViewsHoC(Disease);
 
 	switch (trialListingPageType) {
 		case 'Disease':
-			dynamicRoutes = (
-				<Routes>
-					<Route path={CodeOrPurlPath()} element={<WrappedDisease />} />
-					<Route path="/*" element={<PageNotFound />} />
-				</Routes>
-			);
+			// If banner images are missing, display the invalid parameters route.
```

Comment:
Do we want an invalid parameters route or do we want to just hide the banner?

Reply:
We want the invalid parameters route. After a discussion with Bryan, it was determined that the app should show that invalid parameters were given if 1) images are missing on a Disease or Intervention page, or 2) if images are given on a Manual page.

Reply:
And to give even more detail to those situations:
1) only if both images are present do we set the routes for Disease or Intervention
2) only if both images are missing do we set the routes for Manual
So if only one of the images or neither is set, Disease and Intervention would fail and set the invalid params route.
If only one of the images or both is set, Manual would fail and set the invalid params route.

I am typing this out also for my own sake, because discrete math is hard.

Per Seyi's comment below, I am updating this logic, but that's the explanation of the required/intended functionality!